security/pkg/nodeagent/cache/secretcache_test.go

```diff
@@ -189,6 +190,19 @@ func TestWorkloadAgentRefreshSecret(t *testing.T) {
 	if retries == 5 {
 		t.Errorf("Cached secret failed to get refreshed, %d", atomic.LoadUint64(&sc.secretChangedCount))
 	}
+
+	key := ConnKey{
+		ProxyID:      testProxyID,
+		ResourceName: testResourceName,
+	}
+	if _, found := sc.secrets.Load(key); found != true {
```

Comment:
You can drop the `!= true` and `!= false` in the `if` conditionals. Replace them with `!found` and `found` respectively.